style.css

```diff
@@ -0,0 +1,479 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
+
+html {
+  font-size: 15px;
+  font-family: 'montserratregular', san-serif;
+}
+
+body {
+  margin: 0 auto;
+  position: relative;
+  min-width: 320px;
+  max-width: 1440px;  
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)